botx/clients/clients/sync_client.py

```diff
@@ -108,19 +108,38 @@ def execute(self, request: HTTPRequest) -> HTTPResponse:
                 headers=request.headers,
                 params=request.query_params,
                 json=request.json_body,
+                data=request.data,
+                files=request.files,
             )
         except httpx.HTTPError as httpx_exc:
             raise BotXConnectError(
                 url=request.url,
                 method=request.method,
             ) from httpx_exc
 
+        headers = dict(response.headers)
+
+        should_process_as_error = (
+            response.status_code in request.should_process_as_error
+        )
+        if (  # noqa: WPS337
+            not response.is_error
+            and not should_process_as_error  # noqa: W503
+            and request.expected_type == ExpectedType.BINARY  # noqa: W503
+        ):
```

Comment:
Use variable for explain expression. Like IS_FILE_RESPONSE. For `not response.is_error and request.expected_type == ExpectedType.BINARY`